parm/config/gefs/config.fcst

```diff
@@ -5,6 +5,8 @@
 
 echo "BEGIN: config.fcst"
 
+export USE_ESMF_THREADING="YES"  # Toggle to use ESMF-managed threading or traditional threading in UFSWM
```

Comment:
Do we want to template it and set it in the hosts files so it is machine-dependent?

Reply:
It is unclear yet if this will/should be machine dependent.